docs/evm_compatibility/json_rpc_api_methods.md

```diff
@@ -1,15 +1,17 @@
 ---
 title: JSON-RPC API Methods
 proofedDate: na
-iterationBy: na
+iterationBy: HB
 includedInSite: true
 approvedBy: na
-comment: 
+comments: TODOs inline 
 ---
 
-*The following tables present the full list of JSON-RPC methods. Each of them is in one of the states:*
+This page presents the full list of JSON-RPC methods. Each of them is in one of three states:
 
-|![done](img/done.ico)| — done|![in progress](img/inprogress.ico) | — in progress|![todo](img/todo.ico) | — to do|
+<!-- todo Andrey can help by providing the deviations fr0m ETH standard methods / Neon equivalent will send 1n Slack todo Neon methods missing -- needs update == Andrey?? -->
+
+|![done](img/done.ico)| — done|![in progress](img/inprogress.ico) | — in progress|![not supported](img/false-copy.png) | — not supported|
```

Comment:
@afalaleev I seem to recall, but failed to make a note >> that all the "in progress" items were supposed to become "not supported". Please can you confirm my memory of this!